src/packs/ranged/Laser Rifle.yml

```diff
@@ -1,16 +1,13 @@
 _id: TBWC7vnGuziN2Icf
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: true
     longReload: false
     max: 20
     suppress: false
     type: typeAPower
     value: 20
-  bundle:
-    amount: 1
-    bundled: false
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.